bin/server.py

```diff
@@ -57,6 +57,7 @@ def initEngine():
 
 app = Flask(__name__)
 app.config["default_engine"] = engine
+app.config["dax.imgserv.default_source"] = "/lsst7/releaseW13EP"
```

Comment:
We should probably come up with a different default. @timj , any suggestions? :)